utils/bank.go

```diff
@@ -105,3 +106,24 @@ func FilteredPaginateAccountBalances(
 
 	return res, nil
 }
+
+// SanitizeCoins takes an unsorted list of coins and sorts them, removes coins with amount zero and combines duplicate coins
+func SanitizeCoins(coins []sdk.Coin) sdk.Coins {
```

Comment:
have you seen the [NewCoins](https://github.com/neutron-org/cosmos-sdk/blob/a02d35388763a73f65cab154a9e546079b73fb05/types/coin.go#L180) func from cosmos-sdk? it does exactly the same according to the comment: removes zero coins and sorts the rest. if you have, any reasons you decided to write your own algorithm for it (I presume there could be some subtle differences)? if you have not, do you think we can reuse it?

Reply:
NewCoins doesn't remove duplicates